src/main/java/com/dataloom/authorization/HazelcastAuthorizationService.java

```diff
@@ -55,10 +63,25 @@
     public HazelcastAuthorizationService(
             HazelcastInstance hazelcastInstance,
             AuthorizationQueryService aqs,
-            EventBus eventBus ) {
+            EventBus eventBus,
+            Neuron neuron ) {
         aces = hazelcastInstance.getMap( HazelcastMap.PERMISSIONS.name() );
         this.aqs = checkNotNull( aqs );
         this.eventBus = checkNotNull( eventBus );
+
+        aces.addEntryListener( (EntryUpdatedListener) event -> {
```

Comment:
You will probably have serialization problems here once you're not running in a single jvm